pod/pod-api-public.yaml

```diff
@@ -4522,7 +4522,71 @@ paths:
         '500':
           description: 'Server error, see response body for further details.'
           schema:
-            $ref: '#/definitions/Error'
+            $ref: '#/definitions/Error'
+  '/dataquery/api/v3/messages/thread':
```

Comment:
- `pod-api-public.yaml` should be modified in SBE first : https://github.com/SymphonyOSF/SBE/blob/dev/swagger/pod-api-client/src/main/swagger/pod-api-paths-public.yaml
-  all endpoints documented here are the ones developed and maintained by PLAT team. In other term, the ones that are listed here : https://developers.symphony.com/restapi/reference
- we should never see endpoints like `/dataquery/...` or `/datafeed/...` which are supposed to be internal only (or used by SFE)

Reply:
Finally, if this endpoint requires crypto : it means that it should be part of the Agent API and not part of the pod API.